src/test/java/org/poolc/api/member/MemberAcceptanceTest.java

```diff
@@ -91,7 +97,23 @@ void updateMemberInfo() {
     }
 
     @Test
-    void adminCanDeleteMember() {
+    void adminDeleteExistingMember() {
+        String accessToken = loginRequest("ADMIN_ID", "ADMIN_PASSWORD")
+                .as(AuthResponse.class)
+                .getAccessToken();
+
+        ExtractableResponse<Response> memberResponse = getMembersRequest(accessToken);
+        Gson gson = new Gson();
+        Type type = new TypeToken<ArrayList<MemberResponse>>() {
+        }.getType();
+        List<MemberResponse> list = gson.fromJson(memberResponse.body().asString(), type);
+        ExtractableResponse<Response> response = deleteMemberRequest(accessToken, list.get(0).getUuid());
```

Comment:
```suggestion
                ExtractableResponse<Response> membersResponse = getMembersRequest(accessToken);
        List<MemberResponse> members = membersResponse.body().jsonPath().getList(".", MemberResponse.class);

        ExtractableResponse<Response> response = deleteMemberRequest(accessToken, members.get(0).getUuid());
```

요렇게 바꿀 수 있겠네요. 그러면 Gson 의존성 제거도 가능하겠네요 (시간나면 의존성 낮출수록 이득인 이유를 생각해보고 찾아봐도 좋을 것 같아요)